content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -88,12 +88,26 @@ Refer to the [Customizable Install with Helm](/docs/setup/install/helm) for comp
 Use the following command to render and apply Istio CNI components and override the default configuration of the
 `logLevel` and `excludeNamespaces` parameters for `istio-cni`:
 
+Create a IstioControlPlane CR yaml locally with your override to install istio, e.g. cni.yaml
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  cni:
+    enabled: true
+  values:
```

Comment:
Instead of using force flag, you can put the values not in the schema in unvalidatedValues for now, and could you please create an issue to fix the schema? Thanks!

Reply:
done